modules/network-observability-tcp-flag-syn-flood.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// network_observability/metrics-alerts-dashboards.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-tcp-flag-syn-flood_{context}"]
+= Detecting SYN flooding using the FlowMetric API and TCP flags
+You can create an `alertRule` to alert for SYN flooding. 
```

Comment:
This says "`alertRule`" but later we have "`AlertingRule` resource". Should it be one of the following?

```suggestion
You can create an alert rule to alert for SYN flooding. 
```

```suggestion
You can create an `AlertingRule` resource to alert for SYN flooding. 
```